### DISCUSSION TITLE
Proje 2 - Online Derleyici Kullanımı

### DISCUSSION THREAD

**Discussion by selinkacmaz** [gusanmaz/BMB311_OS_2023#10] [Q&A]
projede çalıştırmamız gereken kodları online bir derleyici üzerinden göstermemiz sorun olur mu?

**Comment by gusanmaz** [comment]
Kodları Linux İşletim Sistemi altında, olmuyorsa sanal makinede çalıştırmanız tercih edilmektedir.

Kodlarınızı çalıştırmak için online derleyici kullanabilirsiniz ama bu durumda proje notunuzda bir kesinti yapılacaktır.
